### DISCUSSION TITLE
PnP Search Results - Have comma in count

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3535] [Q&A] [answered]
Hello all,

My search results number/count is large (as shown in the image). Is there a way to have commas in these numbers so it's easy readable?

![image](https://github.com/microsoft-search/pnp-modern-search/assets/159083804/84ffaa97-7b90-4cb1-9a78-6bee9147e412)

**Comment by kasperbolarsen** [comment]
It might be possible but it will require development,  and should handle multiple ways to display thousands. A comma will work for you but where I live we use a dot for that purpose

**Reply by ellanrk** [reply]
@kasperbolarsen  - Even for a dot, does it need development work?

**Reply by kasperbolarsen** [reply]
Yes

**Reply by ellanrk** [reply]
Oh no. Thanks for the details!

**Reply by kasperbolarsen** [reply]
in TemplateService.TS this helper has to be updated in order show either comma or dots

// Return the search result count message
        // Usage: {{getCountMessage totalRows keywords}} or {{getCountMessage totalRows null}}
        this.Handlebars.registerHelper("getCountMessage", (totalRows: string, inputQuery?: string) => {
            let countResultMessage;
            if (inputQuery) {
                const safeQuery = this.Handlebars.escapeExpression(inputQuery);
                countResultMessage = Text.format(strings.HandlebarsHelpers.CountMessageLong, totalRows, safeQuery);
            } else {
                countResultMessage = Text.format(strings.HandlebarsHelpers.CountMessageShort, totalRows);
            }

            return new this.Handlebars.SafeString(countResultMessage);
        });

**Comment by wobba** [comment, marked as answer]
Try this helper https://github.com/helpers/handlebars-helpers?tab=readme-ov-file#addcommas

`{{addCommas @root.data.totalItemsCount}}`

or

`{{getCountMessage (addCommas @root.data.totalItemsCount) @root.inputQueryText}}`

**Comment by kasperbolarsen** [comment]
I tried to tinker with the Cards layout and 
replaced 
    {{#if @root.properties.showResultsCount}}
                <div class="template--resultCount">
                    <label class="ms-fontWeight-semibold">{{getCountMessage @root.data.totalItemsCount @root.inputQueryText}}</label>
                </div>
            {{/if}}
with
    {{#if @root.properties.showResultsCount}}
                <div class="template--resultCount">
<label class="ms-fontWeight-semibold">{{getCountMessage  (addCommas @root.data.totalItemsCount) @root.inputQueryText}}</label>
                </div>
            {{/if}}

And can confirm it works :-)

**Reply by ellanrk** [reply]
Thanks!
